### DISCUSSION TITLE
Help for a newbie

### DISCUSSION THREAD

**Discussion by murateri75** [Ysurac/openmptcprouter#2941] [General]
Hello 
Just installed OpenMPTCProuter version 0.59.1 from your download repository (Raspberry PI 2B/3B/3B+/4B (32-bits)  - 64-bits does not work)
The OS booting ok
No sshd installed (/usr/sbon) 
Try to install from source but tar : invalid tar magic
Why is there no sshd installed on this version (wrong version) ?
Thanks in advance

**Comment by Ysurac** [comment]
Hi,
64bits image is working, try another tool to write the image like https://rufus.ie/ as a problem with etcher was already reported.
dropbear is used instead of openssh and server is started by default.

**Comment by murateri75** [comment]
Thanks will try that

**Comment by murateri75** [comment]
Hummm was using image for 3B/3B+ but I checked mine is 4B :p 
Running now but still no sshd

**Reply by Ysurac** [reply]
As I said there is no sshd but dropbear that provide SSH server. Router is available on port 22 ip 192.168.100.1. If you can ping it you should be able to SSH to it.

**Reply by murateri75** [reply]
Okay sorry I did not catch no sshd

**Comment by murateri75** [comment]
So now working on eth0 
Thanks an sorry for misundertsanding
